tests/unittests/tiglWingCSTProfile.cpp

```diff
@@ -242,6 +289,23 @@ TEST(CSTApprox, hypersonicAirfoil)
             devmax=deviation;
         }
     }
-    // approximation should be exact since we require only degree 2 spline
-    ASSERT_NEAR(0.0, devmax, 1e-12);
+
+    if (algo == tigl::CCSTCurveBuilder::Algorithm::Piecewise_Chebychev_Approximation) {
+        // approximation should be exact since we require only degree 2 spline

```

Comment:
Why is this so?

Reply:
Not sure. The comment was there prior to this PR. I would have to check, but it really is implementation dependent. While the Chebychev-Algo might yield the exact solution, another algorithm that approaches the optimum iteratively might not.

Reply:
Ah ok, I just thought I might have overseen some "obvious" property of the CST curve in this case.